auth/signature_test.go

```diff
@@ -0,0 +1,91 @@
+package auth
+
+import (
+	"errors"
+	"testing"
+)
+
+const detaExampleAccessToken = "aaaaaaa_xxxxxxxxxxxxxxxx"
+const detaExampleSignature = "v0=aaaaaaa:9369e755d35ea272895cab8546d745107f7b21e6e87e1324e1f85627c04934dd"
+
+func TestCalcSignature_EarlyReturnNotV0(t *testing.T) {
```

Comment:
better to do table driven tests here as well

Reply:
Gotcha, thanks!